apps/backend/src/libs/modules/server-application/libs/types/server-application-route-parameters.type.ts

```diff
@@ -1,5 +1,6 @@
 import { type FastifyReply, type FastifyRequest } from "fastify";
 
+import { type APIPreHandler } from "~/libs/modules/controller/libs/types/api-pre-handler.type.js";
```

Comment:
this type shoud be re-exported from `~/libs/modules/controller/controller.js`, we can't access module internal files